includes/class-wc-payments.php

```diff
@@ -891,6 +893,7 @@ public static function ajax_init_platform_checkout() {
 
 		$platform_checkout_host = defined( 'PLATFORM_CHECKOUT_HOST' ) ? PLATFORM_CHECKOUT_HOST : 'http://host.docker.internal:8090';
 		$url                    = $platform_checkout_host . '/wp-json/platform-checkout/v1/init';
+		$store_api_url          = Package::container()->get( RoutesController::class )->get( 'cart' )->get_namespace();
```

Comment:
I recall working on [this code](https://github.com/Automattic/woocommerce-payments/blob/abbe046497c841633325d7b8dc57b6f4c75493b2/tests/unit/admin/test-class-wc-rest-payments-settings-controller.php#L523) where depending on the version of WC Blocks installed, the availability of `Package::container()` (or its methods) would differ. We should also account for that here.

Maybe try installing versions of WC Blocks earlier than 2.6.0 and checking if this PR works?

Once we get that ironed out, we should also make sure that even if `Package::container()` is present:
* is `Package::container()->get( RoutesController::class )` present?
* is `Package::container()->get( RoutesController::class )->get( 'cart' )` present?
* is `Package::container()->get( RoutesController::class )->get( 'cart' )->get_namespace()` present?